docs/releases/5.0.7.txt

```diff
@@ -9,4 +9,6 @@ Django 5.0.7 fixes several bugs in 5.0.6.
 Bugfixes
 ========
 
-* ...
+* Fixed a bug when calling :meth:`~django.db.models.Model.get_FOO_display` for
+  a ``GeneratedField`` with defined ``choices`` on
+  :attr:`~django.db.models.GeneratedField.output_field` (:ticket:`35566`).
```

Comment:
This is no niche that I would treat this as a new feature (ticket suggests the same), and we don't backport new features.

Reply:
I know we don't backport new features, I wasn't sure whether my original assessment that this is a new feature and not a bug was accurate. 
So I appreciate having another opinion here, thank you - updated 👍